src/lib/storage/chunk.hpp

```diff
@@ -143,11 +143,18 @@ class Chunk : private Noncopyable {
   /**
    * If a chunk is sorted in any way, the order (Ascending/Descending/AscendingNullsFirst/AscendingNullsLast) and
    * the ColumnID of the segment by which it is sorted will be returned.
-   * This is currently only taken advantage of in the ColumnVsValueScan. See #1519 for more details.
+   * This is currently taken advantage of in, e.g., the ColumnVsValueScan. See #1519 for more details.
    */
   const std::optional<std::pair<ColumnID, OrderByMode>>& ordered_by() const;
   void set_ordered_by(const std::pair<ColumnID, OrderByMode>& ordered_by);
 
+  /**
+   * If a chunk is value clustered in any way, the clustering type and the ColumnID of the segment by which it is
+   * clustered will be returned.
```

Comment:
Could you explain what "clustered" means and how it is different from ordering? Also, where does the "type" of clustering come into play? I can't see it in the signature.